vlib/v/checker/tests/assign_blank_none_err.out

```diff
@@ -0,0 +1,5 @@
+vlib/v/checker/tests/assign_blank_none_err.vv:2:2: error: cannot assign none to blank `_` identifier
+    1 | fn main() {
+    2 |     _ = none
+      |     ^
+    3 | }
```

Comment:
Add new line at the end

Reply:
@Delta456 why is a newline required here?

Reply:
It's only needed in a .out file if it helps match against the test output.